galaxy_ng/app/access_control/access_policy.py

```diff
@@ -179,3 +180,7 @@ class ContainerNamespaceAccessPolicy(AccessPolicyBase):
 
 class ContainerRegistryRemoteAccessPolicy(AccessPolicyBase):
     NAME = 'ContainerRegistryRemoteViewSet'
+
+
+class RemoteSyncAccessPolicy(AccessPolicyBase):
+    NAME: 'SyncRemoteView'
```

Comment:
This should be `NAME = `.

The name here corresponds to items in the dictionary in `app/access_control/statements/standalone.py` not the name of the viewset.